### DISCUSSION TITLE
Can't index new images on RPI4 - docker  -- SOLUTION INSIDE --

### DISCUSSION THREAD

**Discussion by kavemang** [photoprism/photoprism#1786] [Q&A] [answered]
I found my password was reset when trying to access photoprism today, upon getting in all images are gone, everything I setup as far as albums and faces is gone. When I try to re-index the originals the container crashes on the 3rd or 4th images with nothing seeming to appear out of the ordinary in the container logs. 

The final lines say: 

```
/entrypoint.sh: line 94:    10 Killed                  "$@"
 PMContainer stopped
```

PS: it looks like photoprism created two new volumes today for its storage and config sections, any idea why it did this/? 
ive since then restored the original volumes and my indexing and albums are back 

**Im finding even with last months version I can no longer index new images without it crashing out with the 10killed or 9 killed message, im at a total loss for what is going on** 

It always crashes out with : 
`/entrypoint.sh: line 94:    10 Killed                  "$@"`

### SOLUTION: I implemented the Cgroup settings here: [https://github.com/photoprism/photoprism/issues/1172](https://github.com/photoprism/photoprism/issues/1172)
**and now indexing is working again, I also noticed photoprism was spreading core load instead of only using one while indexing after setting these options, that said I have absolutely no idea why I had to do this to get it working all of the sudden, going to leave this up for a bit to see if anyone can comment**

**Comment by lastzero** [comment, marked as answer]
SOLUTION: I implemented the Cgroup settings here: #1172
and now indexing is working again, I also noticed photoprism was spreading core load instead of only using one while indexing after setting these options, that said I have absolutely no idea why I had to do this to get it working all of the sudden, going to leave this up for a bit to see if anyone can comment

**Comment by raghubhaskar** [comment]
PP hit 30K images during scan/index and I got this error. I increased the RAM size from 2GB to 4GB and it is working.
